components/infobox/wikis/ageofempires/infobox_team_custom.lua

```diff
@@ -0,0 +1,60 @@
+---
+-- @Liquipedia
+-- wiki=ageofempires
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local GameLookup = require('Module:GameLookup')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+local Template = require('Module:Template')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local Team = Lua.import('Module:Infobox/Team', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+
+local CustomTeam = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+function CustomTeam.run(frame)
+	local team = Team(frame)
+
+	-- Automatic achievements
+	team.args.achievements = Template.expandTemplate(frame, 'Team achievements', {team.pagename, aka = team.args.aka})
```

Comment:
Could directly use `Module:Achievements in Infoboxes', no?

Reply:
please less template expands is always a good idea